docs/extra/logs-collect.md

```diff
@@ -0,0 +1,23 @@
+# Collecting logs for Support
+
+To help troubleshoot issues, obtain the logs from your Codacy instance and send them to Codacy's Support:
+
+1.  Collect logs from your Codacy instance using Fluentd:
+
+    ```bash
+    --set fluentdoperator.enabled=true
+    ```
+
+    After executing this command the Fluentd daemonset will send the logs to MinIO, which is also installed by the Codacy chart.
+
+2.  Download the logs by running the following command locally, replacing `<namespace>` with the namespace in which Codacy was installed:
+
+    ```bash
+    bash <(curl -fsSL https://raw.githubusercontent.com/codacy/chart/master/docs/extra/extract-codacy-logs.sh) -n <namespace>
+    ```
+
+    The script compresses the logs in an archive file with the name `codacy_logs_<timestamp>.zip`.
+    
+    You can also [download the script](extract-codacy-logs.sh) to run it manually.
+
+3.  Send the archive file containing the logs to Codacy's support team for analysis.
```

Comment:
Maybe add `support@codacy.com`

Reply:
I don't know if we have other documentation for this process, but it might be possible that the log file is too big for email and that the client needs to agree on a way to send us the logs. Not sure if we should add that here.

Reply:
Both are good suggestions.

I'm [figuring out](https://codacy.slack.com/archives/CQESXTUHW/p1585045849010900) what is the usual procedure for customers to send bigger files to support.